.github/workflows/uploadfiles_app-backend-zkrnwcgqiomq6.yml

```diff
@@ -0,0 +1,76 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+# More info on Python, GitHub Actions, and Azure App Service: https://aka.ms/python-webapps-actions
+
+name: Build and deploy Python app to Azure Web App - app-backend-zkrnwcgqiomq6

```

Comment:
You can remove most of the steps and call `azd deploy` in one step.  See this workflow for an example: https://github.com/Azure-Samples/todo-python-mongo/blob/main/.github/workflows/azure-dev.yml

Reply:
nice @jongio , Thanks for sharing, will update